### ISSUE TITLE
Missing strings in application window

### ISSUE THREAD

**Issue by Zlopez** [flathub/com.nextcloud.desktopclient.nextcloud#115]
I'm using Fedora Silverblue 38 and I'm currently experiencing this weird issue when I don't see most of the strings in the main window. I'm also unable to click on the items in dropdown menu.
![image](https://github.com/flathub/com.nextcloud.desktopclient.nextcloud/assets/6943409/134e0eec-0357-46f2-af45-c2ab795684a0)

**Comment by mrkskwsnck** [comment]
I am experiencing the very same issue on Vanilla OS

```
$ cat /etc/os-release 
PRETTY_NAME="VanillaOS 22.10 all"
NAME="VanillaOS"
VERSION_ID="22.10"
VERSION="22.10 all"
VERSION_CODENAME="kinetic"
ID=ubuntu
ID_LIKE=debian
HOME_URL="https://github.com/vanilla-os"
SUPPORT_URL="https://github.com/vanilla-os"
BUG_REPORT_URL="https://github.com/vanilla-os"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
UBUNTU_CODENAME="kinetic"
```

running GNOME Shell with either X11 or Wayland

```
$ gnome-shell --version
GNOME Shell 43.1

$ printenv XDG_SESSION_TYPE
wayland
```

with Nextcloud sync client as flatpak

```
$ flatpak list | grep -i next
Nextcloud Desktop	com.nextcloud.desktopclient.nextcloud	3.9.1	stable	user
```

There seems to be no way to access the settings 🤷🏼 Or is there any way around to get to the settings, though?

**Comment by mrkskwsnck** [comment]
Issue https://github.com/flathub/com.nextcloud.desktopclient.nextcloud/issues/109 might be related…

**Comment by julianfairfax** [comment]
I am also having this issue on Debian Trixie.

**Comment by BigmenPixel0** [comment]
I have the same problem on ArchLinux with GNOME.

**Comment by ejgr-mtsiw** [comment]
> Same issue. Here is my output:
> 
> ```shell
> $ flatpak run com.nextcloud.desktopclient.nextcloud
> QSocketNotifier: Can only be used with threads started with QThread
> qt.qpa.qgnomeplatform: Could not find color scheme  ""
> 
> 
> ** (nextcloud:2): WARNING **: 10:43:39.876: atk-bridge: get_device_events_reply: unknown signature
> ```
> 
> EDIT: Added the last line, which comes up after a couple of minutes.

I have exactly the same output and result: blank menu, and wrong colours. The last menu icon that opens the Nextcloud apps is also missing.

System:
OS: Fedora Silverblue Linux 38 x86_64
gnome-shell --version -> GNOME Shell 44.3
printenv XDG_SESSION_TYPE -> wayland

![imagem](https://github.com/flathub/com.nextcloud.desktopclient.nextcloud/assets/59804682/9886f158-45dc-487d-9881-ae2273b7d458)

**Comment by HarryEuro** [comment]
Same problem on Ubuntu:
- Ubuntu 22.04.3 LTS
- GNOME 42.9
- Wayland

**Comment by ikke-t** [comment]
Also on Fedora Silverblue 39 (beta)

**Comment by sallyhaj** [comment]
I am facing the same problem in Debian 12 Bookworm.

**Comment by mrkskwsnck** [comment]
> I am facing the same problem in Debian 12 Bookworm.

Me 2. I fixed this very issue by switching ALL THE Debian/GNU/Linux repos to testing (a.k.a. trixie) 🤷🏼‍♂️

**Comment by p-fruck** [comment]
Note that you should be able to fix the issue by overriden the env variables and setting `QT_QPA_PLATFORMTHEME=kde` as shown in #130

**Comment by p-fruck** [comment]
Thanks, but unfortunately the PR fixing this issue is still open (#130). #131 was just an update of the KDE runtime version

**Comment by tilosp** [comment]
yes sorry, github auto-closed when i merged the pr

**Comment by julianfairfax** [comment]
Downgrading to `a3a30f66c8173deb6efeedfe762292c4d8b8241d16735fc73efaf1353b4b2734` "fixes" this issue. I did it this way:
```
flatpak install com.nextcloud.desktopclient.nextcloud --no-deps
flatpak remote-info flathub com.nextcloud.desktopclient.nextcloud --log
sudo flatpak update com.nextcloud.desktopclient.nextcloud --commit=a3a30f66c8173deb6efeedfe762292c4d8b8241d16735fc73efaf1353b4b2734
sudo flatpak mask com.nextcloud.desktopclient.nextcloud
```

However, since #131, running `flatpak update` now wants to install the newer runtime that's not used by the downgraded version:
```
Suche nach Aktualisierungen …

Required runtime for com.nextcloud.desktopclient.nextcloud/x86_64/stable (runtime/org.kde.Platform/x86_64/5.15-23.08) found in remote flathub
Möchten Sie es installieren? [Y/n]: 


        KENNUNG                                             Zweig      Op Gegenstelle Herunterladen
 1.     org.kde.Platform.Locale                             5.15-23.08 i  flathub     < 377.8 MB (teilweise)
 2.     org.kde.PlatformTheme.QGnomePlatform                5.15-23.08 i  flathub      < 10.6 MB
 3.     org.kde.WaylandDecoration.QAdwaitaDecorations       5.15-23.08 i  flathub     < 930.4 KB
 4.     org.kde.WaylandDecoration.QGnomePlatform-decoration 5.15-23.08 i  flathub      < 11.1 MB
 5.     org.kde.Platform                                    5.15-23.08 i  flathub     < 335.1 MB

Mit diesen Änderungen an der Systeminstallation fortfahren? [Y/n]: 
```

If you actually install the unneeded runtimes, running `flatpak uninstall --unused` uninstalls them again.

It would be nice if this issue could be fixed somehow, because using an old version is not good for security, or just having new features. But, in addition, does anyone know how I can stop flatpak from wanting to install the new runtime?

**Comment by p-fruck** [comment]
I don't know if I'm missing something. Everything required to fix the issue is defining one environment variable. I've prepared [this PR](https://github.com/flathub/com.nextcloud.desktopclient.nextcloud/pull/130) which fixes this issue, but unfortunately it is still unmerged. If this PR would be merged, we'd have a fixed nextcloud client with the latest runtime versions.

**Comment by julianfairfax** [comment]
See a comment from in #139 for the proper solution to this issue:

> This reverts commit daf49adeff317e44d997941351a4f2a75dbb5562, reversing changes made to 646b3d378a7780bb5cfc52ef259cda9777d4ff6a.

> The previous merge request "fixed" #115 by "removing GNOME theming". It did this in the wrong way though, by forcing other platform-specific theming.

> Unfortunately, despite its deprecation, Flatpak still installs GNOME theming for Qt apps when a Qt app is installed. Once I figure out where to propose that this be changed, I will do.

> However, in the meantime, forcing other platform-specific theming is not the solution, if the GNOME theming breaks Nextcloud.

> The current solution must be implemented by Flatpak, and, until then, can be achieved by uninstall the GNOME theming if it's already installed, and preventing it from be installed in the future:
> ```
> flatpak uninstall org.kde.KStyle.Adwaita org.kde.PlatformTheme.QGnomePlatform org.kde.WaylandDecoration.QAdwaitaDecorations org.kde.WaylandDecoration.QGnomePlatform-decoration
> 
> sudo flatpak mask org.kde.KStyle.Adwaita org.kde.PlatformTheme.QGnomePlatform org.kde.WaylandDecoration.QAdwaitaDecorations org.kde.WaylandDecoration.QGnomePlatform-decoration
> ```

Tl;dr: run this:
```
flatpak uninstall org.kde.KStyle.Adwaita org.kde.PlatformTheme.QGnomePlatform org.kde.WaylandDecoration.QAdwaitaDecorations org.kde.WaylandDecoration.QGnomePlatform-decoration

sudo flatpak mask org.kde.KStyle.Adwaita org.kde.PlatformTheme.QGnomePlatform org.kde.WaylandDecoration.QAdwaitaDecorations org.kde.WaylandDecoration.QGnomePlatform-decoration
```

**Comment by MisterMX** [comment]
The above solution does not work for me because I need the mentioned packages in other apps.

However, I was able to circumvent this issue on Gnome by setting the environment variable `QT_QPA_PLATFORMTHEME=kde`. Now it doesn't fit visually but at least I can access the settings.

**Comment by hellpe** [comment]
I'm still having this issue using version 3.12. Should I wait for a newer release on order to benefit from the PR changes?
